src/js/WinningLotto.js

```diff
@@ -0,0 +1,31 @@
+import { ERROR_MESSAGE } from './constants.js';
+import { LottoNumber } from './LottoNumber.js';
+
+export class WinningLotto {
+  lottoNumber;
+  bonusNumber;
+  constructor({ lottoNumber, bonusNumber }) {
+    this.lottoNumber = lottoNumber;
+    this.bonusNumber = bonusNumber;
+    this.validate();
+  }
+
+  validate() {
+    if (this.isInValidWinningNumbers([...this.lottoNumber, this.bonusNumber])) {
+      throw Error(ERROR_MESSAGE.EMPTY_RANGE);
+    }
+
+    if (this.isDuplicateNumbers([...this.lottoNumber, this.bonusNumber])) {
+      throw Error(ERROR_MESSAGE.DUPLICATED);
+    }
+  }
+
+  isInValidWinningNumbers(winningNumbers) {
+    return winningNumbers.some((number) => new LottoNumber(number).isInValid);
+  }
+
+  isDuplicateNumbers(numbers) {
+    const numberCollection = new Set(numbers);
+    return numbers.length !== numberCollection.size;
+  }
```

Comment:
```suggestion
  isDuplicateNumbers() {
    return this.lottoNumbers.includes(this.bonusNumber);
  }
```
지금 이 WinningLotto 라는 클래스가 Lotto의 역할까지 대신 수행중인 것 같아요.
검증된 lottoNumbers가 WinningLotto에 전달되어야 하는게 아닐까요?
여기서 Lotto를 분리하면 WinningLotto의 역할이 더 명시적이지 않을까요?